rtl/eclrtl/rtlrecord.hpp

```diff
@@ -250,6 +251,7 @@ class ECLRTL_API RtlRecord
     const RtlFieldInfo * const * originalFields;
     const RtlRecord **nestedTables;
     const char **names;
+    const char **xpaths;
```

Comment:
I don't think this is used.

Reply:
How are xpaths within nested records supported?  Does it need similar logic to the way names are exapanded?

Reply:
I added a mechanism similar to names, then decided not needed and removed - but forgot to remove this member.
It's not needed, I think, because of the way the fetcher walks nested datasets.

Reply:
Actually it was needed... => resolve these comments